src/api/common/schemas/get-paths.json

```diff
@@ -6,7 +6,23 @@
     "type": "object",
     "properties": {
       "source": {"$ref": "adjustment"},
-      "destination": {"$ref": "adjustment"},
+      "destination": {
+        "type": "object",
```

Comment:
Anyway, the only real objection I had with this PR was using sourceamount
counterparty for the dest amount when unspecified iirc. Which is fixed. I
don't want to hold you up.